.travis.yml

```diff
@@ -0,0 +1,18 @@
+sudo: false
+language: python
+install: pip install tox
+matrix:
+  include:
+    - python: "2.6"
+      env: TOX_ENV=py26
+    - python: "2.7"
+      env: TOX_ENV=py27
+    - python: "3.6"
+      env: TOX_ENV=py36
+    - python: "2.7"
+      env: TOX_ENV=static
+    - python: "2.7"
+      env: TOX_ENV=cov
+    - python: "2.7"
+      env: TOX_ENV=cov-travis
```

Comment:
'cov-travis' makes 'cov' redundant. The html report generated by 'cov' is anyways unusable in CI job. So should only keep 'cov-travis'

Reply:
That's a good point, @rohanpm. We really don't need to run tests on the same version multiple times.